packages/gatsby/src/bootstrap/index.js

```diff
@@ -165,7 +165,9 @@ module.exports = async (args: BootstrapArgs) => {
   const flattenedPlugins = await loadPlugins(config, program.directory)
   activity.end()
 
-  const pluginsStr = flattenedPlugins.map(p => `${p.name}@${p.version}`)
+  // Multiple occurrences of the same name-version-pair can occur,
+  // so we report an array of unique pairs
+  const pluginsStr = _.uniq(flattenedPlugins.map(p => `${p.name}@${p.version}`))
```

Comment:
I don't think we need `_uniq` for this...?

```suggestion
  const pluginsStr = [...new Set(flattenedPlugins.map(p => `${p.name}@${p.version}`))]
```

(While there are more efficient ways the worst-case number of plugins won't warrant going that far).

Reply:
Sure that works for me.

Just so I know in the future, are we purposefully avoiding using lodash? I figured that's the way to go since it's widely used and is required in this file specfically.

Reply:
I'm not against using lodash. I do think that we should try to do stuff with native built-ins as much as possible as long as we're not going "out of our way" just not to use it. So like, if it adds 10 lines of code then it's probably not worth it.
In this case I think it saves many ops of underwater lodash execution :)